build.zig

```diff
@@ -4,7 +4,13 @@ pub fn build(b: *std.Build) void {
     const optimizeOpt = b.standardOptimizeOption(.{});
     const targetOpt = b.standardTargetOptions(.{});
 
+    const datetime = b.createModule(.{ .root_source_file = b.path("src/datetime.zig") });
```

Comment:
Why is it needed?

Reply:
Because the serialize module reuses the `Date`, `Time` and `DateTime` types.